functions-iterators/Homework4.JS

```diff
@@ -0,0 +1,120 @@
+// PART 1: Big or Small String?
+function bigOrSmallString (string){
+
+    if ( string.length > 10) {
+    return "This word is loooooong!" }
+    else {
+    return `This word is short 😬`;
+    }
+}
+// PART 2: Odd or Even String Length?
+
+function oddOrEvenString(string){
+    if (string.length % 2 == 0){
+      return "This string length is TOTALLY even!";
+    } else {
+      return "This string's length is odd";
+    }
+  }
+
+  // PART 3: Median
+
+ var num = [30, 35, 45, 1, 5];
+function plural(array1){
```

Comment:
I don't think you understand this part, I can tell from the function name if you need some further explanation please don't hesitate to reach out